packages/packages/commonwell-sdk/src/models/certificates.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+import { linkSchema } from "./link";
+
+export const certificateSchema = z.object({
+  startDate: z.string(),
+  endDate: z.string(),
+  expirationDate: z.string(),
+  thumbprint: z.string(),
```

Comment:
are these actually required? every time I create certs in the portal, I don't actually specify these values